draft-ietf-ppm-dap.md

```diff
@@ -459,11 +459,10 @@ struct {
 } Interval;
 
 /* A nonce used to uniquely identify a report in the context of a DAP task. It
-includes the time at which the report was generated and a random, 64-bit
-integer. */
+includes the timestamp of the current batch and a random 16-byte value. */
 struct {
   Time time;
```

Comment:
What about moving `time` into `Report`? Then you could modify Report by changing the type of `nonce` to `uint8 [16]`.

Reply:
That would be another way to address the name issue, yeah. But we can do that separately.